test/TestServerProjectsLowLevel/llc_update1/Custom/ParamsClient.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+
+namespace llc_update1_LowLevel
+{
+    /// <summary> The Params service client. </summary>
+    [CodeGenSuppress("PostParametersAsync", typeof(RequestContent), typeof(RequestContext))]
+    [CodeGenSuppress("PostParameters", typeof(RequestContent), typeof(RequestContext))]
+    [CodeGenSuppress("CreatePostParametersRequest", typeof(RequestContent), typeof(RequestContext))]
+    public partial class ParamsClient
+    {
+        /// <summary> POST a JSON or a JPEG. </summary>
+        /// <param name="content"> The content to send as the body of the request. </param>
+        /// <param name="contentType"> Upload file type. Allowed values: &quot;application/json&quot; | &quot;image/jpeg&quot;. </param>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="content"/> is null. </exception>
+#pragma warning disable AZC0002
+        public virtual async Task<Response> PostParametersAsync(RequestContent content, ContentType contentType, RequestContext context = null)
```

Comment:
I believe we want these methods for testing purposes, I wonder why don't we add this method in testserver repo in the [swagger](https://github.com/Azure/autorest.testserver/blob/main/swagger/llc_update1.json) so that other languages can also leverage?